services/API-service/src/api/event/event-pcode.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { Entity, Column, Check, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity('event_pcode', { schema: 'IBF-pipeline-output' })
+export class EventPcodeEntity {
+  @PrimaryGeneratedColumn()
+  public id: number;
+
+  @Column({ nullable: true })
+  public pcode: string;
+
+  @Column({ type: 'timestamp', nullable: true })
+  public start_date: Date;
+
+  @Column({ type: 'timestamp', nullable: true })
+  @Check(`"start_date" < "end_date"`)
+  public end_date: Date;
```

Comment:
`eventEndDate` instead of `end_date`

use [camelCase](https://news.ycombinator.com/item?id=6777600)